src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return String(a) > String(b);
+  }) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => String(a) > String(b)) 
  {
```

You may use this callback without 'return' here